### ISSUE TITLE
Fix handling of LOCALEDIR in two *.c files

### ISSUE THREAD

**Issue by danfe** [rhash/RHash#114]
Two `*.c` files reference `LOCALEDIR` but do not correctly handle it as set by the `configure` script, which breaks NLS support on systems where it is not `/usr/share/locale` (e.g., *BSD).  Please consider the following patch:
```patch
--- configure.orig
+++ configure
@@ -883,6 +883,7 @@ WARN_CFLAGS = $WARN_CFLAGS
 ADDCFLAGS   = $BUILD_EXTRA_CFLAGS
 ADDLDFLAGS  = $BUILD_EXTRA_LDFLAGS
 CONFCFLAGS  = -DSYSCONFDIR=\\"$INSTALL_SYSCONFDIR\\"
+LOCALECFLAGS = -DLOCALEDIR=\\"$INSTALL_LOCALEDIR\\"
 CFLAGS  = $RHASH_DEFINES \$(OPTFLAGS) \$(WARN_CFLAGS) \$(ADDCFLAGS)
 LDFLAGS = \$(OPTLDFLAGS) \$(ADDLDFLAGS) $GETTEXT_LDFLAGS
 BIN_STATIC_LDFLAGS = \$(LDFLAGS) $(join_params $LD_STATIC $OPENSSL_LDFLAGS)
--- Makefile.orig
+++ Makefile
@@ -212,11 +212,11 @@ parse_cmdline.o: parse_cmdline.c parse_cmdline.h commo
 rhash_main.o: rhash_main.c rhash_main.h calc_sums.h common_func.h file.h \
  hash_check.h file_mask.h find_file.h hash_print.h hash_update.h \
  parse_cmdline.h output.h win_utils.h librhash/rhash.h
-	$(CC) -c $(CFLAGS) $< -o $@
+	$(CC) -c $(CFLAGS) $(LOCALECFLAGS) $< -o $@
 
 win_utils.o: win_utils.c win_utils.h common_func.h file.h parse_cmdline.h \
  rhash_main.h
-	$(CC) -c $(CFLAGS) $< -o $@
+	$(CC) -c $(CFLAGS) $(LOCALECFLAGS) $< -o $@
 
 dist/rhash.1.win.html: dist/rhash.1 dist/rhash.1.win.sed
 	sed -f dist/rhash.1.win.sed dist/rhash.1 | rman -fHTML -roff | \
```

**Comment by rhash** [comment]
Thanks! The fix has been commited with slight changes as c4f0c8609137f04eabaa0919b2f314624adc2f03.

**Comment by rhash** [comment]
The fix is included into **RHash v1.4.0**.
